lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb

```diff
@@ -2,14 +2,26 @@
   <div class="center spacious content-block">
     <h1><%= @landing_info['title'].html_safe %></h1>
     <p><%= raw @landing_info['subtitle'].html_safe %></p>
-  </div>
+    <p>
+      <!-- This is the buttons group area-->
+      <strong>Jump to: </strong>
+      <% @landing_info['top-section'].each do |product| %>
+        <a href="#<%=product.keys.first.parameterize %>"  class='Vlt-btn Vlt-btn--blue Vlt-btn--app Vlt-btn--small'><%= product.keys.first %></a>
+      <%end %>
 
-  <div>
+      <% @landing_info['bottom-section'].each do |product| %>
+        <% product[product.keys.first].each do |cards| %>
+          <a href="#<%=cards.keys.first.parameterize %>" class='Vlt-btn Vlt-btn--blue Vlt-btn--app Vlt-btn--small'><%= cards.keys.first %></a>
+        <%end%>
+      <%end%>
+    </p>
+  </div>
+<div>
 
 <!-- TOP section -->
     <% @landing_info['top-section'].each do |product| %>
       <hr class="hr--tall Vlt-gradient--blue-to-pink"></hr>
-      <h2><%= product.keys.first %></h2>
+      <h2 id="<%= product.keys.first.parameterize  %>"><%= product.keys.first %></h2>
```

Comment:
What do you think about adding the ID property to the `<hr>` tag just above each title so it will have more space when the user click on a 'jump to' button

```
<hr id="<%= product.keys.first.parameterize %>" class="hr--tall Vlt-gradient--blue-to-pink"></hr>

Reply:
changed it and it does give it more space.